app/src/main/java/starprogrammers/Reservation.java

```diff
@@ -55,6 +63,10 @@ public class Reservation {
       this.roomNumber = roomNumber;
       this.checkIn = convertLocalDate(checkIn);
       this.checkOut = convertLocalDate(checkOut);
+      period = Period.between(checkIn, checkOut);
+      
+      id++; // each new reservation created increases id by 1
```

Comment:
https://www.w3schools.com/mysql/mysql_autoincrement.asp
We should let the database handle it because if we restart our program that counter will be gone.